mm2src/coins/eth.rs

```diff
@@ -2733,15 +2761,15 @@ impl EthCoin {
             let tx_from_rpc = match tx_from_rpc {
                 Some(t) => t,
                 None => {
-                    return MmError::err(ValidatePaymentError::UnexpectedPaymentState(format!(
+                    return MmError::err(ValidatePaymentError::TxFromRPCError(format!(
```

Comment:
`TxFromRPCError` seems hard to read for me

Reply:
I removed this and used `InvalidRpcResponse` instead.

Reply:
thanks